### ISSUE TITLE
Create javascript controller

### ISSUE THREAD

**Issue by JavaCakez** [thebeansgroup/CodingStyleGuideEngine#24]

**Comment by adtaylor** [comment]
Just want to set expectation around this section. Completely do not expect you to write the docs for the JS repos. Just have this section working so that it can consume repos like Yaks.

**Comment by JavaCakez** [comment]
By consume do you mean a git submodule and reading the README into a partial?

**Comment by adtaylor** [comment]
Yes. Basically what we previously discussed. It needs to be fully working but just don't worry about the actual content of the repos.
